Framework/Core/include/Framework/AnalysisDataModel.h

```diff
@@ -400,6 +400,62 @@ DECLARE_SOA_TABLE(Run2V0s, "AOD", "RUN2V0", run2v0::BCId,
                   run2v0::BBFlag, run2v0::BGFlag);
 using Run2V0 = Run2V0s::iterator;
 
+// ---- MC tables ----
+
+namespace mcvtx
+{
+DECLARE_SOA_INDEX_COLUMN(BC, bc);
+DECLARE_SOA_COLUMN(GeneratorsID, generatorsID, short);
+DECLARE_SOA_COLUMN(X, x, float);
+DECLARE_SOA_COLUMN(Y, y, float);
+DECLARE_SOA_COLUMN(Z, z, float);
+DECLARE_SOA_COLUMN(T, t, float);
+DECLARE_SOA_COLUMN(Weight, weight, float);
+DECLARE_SOA_COLUMN(NProduced, nProduced, int);
```

Comment:
What is nProduced?

Reply:
Number of produced particles in this collision.

Reply:
And we need to store this in addition to the MC particle array?

Reply:
We can remove it, the information can be extracted from the mcparticle table. The initial purpose was fast navigation.